docs/content/en/docs/reference/changelog.md

```diff
@@ -3,12 +3,35 @@ linkTitle: "Changelog"
 weight: 55
 ---
```

Comment:
I'd like to add a link to this page to the menu bar. I think changing the header to look like this would work:
`title: "What's New?"
linkTitle: "What's New?"
weight: 55
menu:
  main:
    weight: 10
---
`

Reply:
So this would replace the two lines of three-dashes and everything in between.